v2/sourcedb-to-spanner/terraform/samples/sharded-migration/globaly-unique-keys/terraform.tfvars

```diff
@@ -0,0 +1,62 @@
+common_params = {
+  on_delete                    = "drain"  # Or "cancel" if you prefer
```

Comment:
[Out of curiosity and might not need a code change],
What happens if a delete is triggered after the job (or jobs) have gone to success state?
Since this is a batch job, it goes to a terminal success state.

Reply:
Good question! On those states it is a `NOOP`.

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/dataflow_flex_template_job.html#note-on-destroy--apply